src/test/scala/uk/gov/hmrc/ct/BoxValidationFixture.scala

```diff
@@ -28,6 +28,21 @@ trait BoxValidationFixture[T <: ComputationsBoxRetriever] extends WordSpec with
   //This can be overridden if mock box retriever calls need to be made
   def setUpMocks(): Unit = Unit
 
+  def testIsPositiveOrNegative(boxId: String, builder: Option[Int] => ValidatableBox[T]) = {
```

Comment:
Type system already checks if the value is an Integer. 
the validator may have a name like: noConstrainsValidation or not exist at all if it is easy to do.